Makefile

```diff
@@ -135,7 +135,7 @@ build-functest:
 
 functest: build-functest
 	@echo "Running ocs developer functional test suite"
-	hack/functest.sh
+	hack/functest.sh $(ARGS)
```

Comment:
Didn't know it can take arguments. Is there an example of how I can make use of available args?